docs/changes.rst

```diff
@@ -333,7 +333,7 @@ Changes:
     - compression: use the 2 bytes for type and level, #6698
     - use version 2 for new archives
     - convert timestamps int/bigint -> msgpack.Timestamp, see #2323
-    - all hardlinks have chunks, maybe chunks_healty, hlid
+    - all hardlinks have chunks, maybe chunks_healthy, hlid
```

Comment:
Some projects don't like changing changelogs, happy to drop this or anything else...

Reply:
changing is fine if it is correcting an error.